spec/index.html

```diff
@@ -3775,6 +3788,150 @@ <h2>Acknowledgements</h2>
+    </li>
+    <li>
+      Carroll, J.J., Bizer, C., Hayes, P., Stickler, P.:
+      Named graphs.
+      Journal of Web Semantics, 3 (4), pp. 247-267,
+      World Wide Web Conference 2005——Semantic Web Track  (2005).
+      <a href="https://doi.org/10.1016/j.websem.2005.09.001">https://doi.org/10.1016/j.websem.2005.09.001</a>.
+    </li>
+    <li>
+      Tummarello, G., Morbidoni, C., Bachmann-Gmür, R., Erling, O.:
+      RDFSync: Efficient Remote Synchronization of RDF Models.
+      In: Aberer, K., Choi, K.-S., Noy, N., Allemang, D., Lee, K.-I., Nixon, L., Golbeck, J., Mika, P., Maynard, D., Mizoguchi, R., Schreiber, G., and Cudré-Mauroux, P. (eds.)
+      The Semantic Web. pp. 537–551.
+      Springer, Berlin, Heidelberg (2007).
+      <a href="https://doi.org/10.1007/978-3-540-76298-0_39">https://doi.org/10.1007/978-3-540-76298-0_39</a>.
+    </li>
+    <li>
+      Oren, E., Delbru, R., Catasta, M., Cyganiak, R., Stenzhorn, H., Tummarello, G.:
+      Sindice.com: a document-oriented lookup index for open linked data.
+      International Journal of Metadata, Semantics and Ontologies. 3, 37–52 (2008).
+      <a href="https://doi.org/10.1504/IJMSO.2008.021204">https://doi.org/10.1504/IJMSO.2008.021204</a>.
+    </li>
+    <li>
+      Fisteus, J.A., Fernández García, N., Sánchez Fernández, L., Delgado Kloos, C.:
+      Hashing and canonicalizing Notation 3 graphs.
+      Journal of Computer and System Sciences. 76, 663–685 (2010).
+      <a href="https://doi.org/10.1016/j.jcss.2010.01.003">https://doi.org/10.1016/j.jcss.2010.01.003</a>.
+    </li>
+    <li>
+      Gross, J.L., Yellen, J., Zhang, P.:
+      Handbook of Graph Theory, Second Edition.
+      Chapman & Hall/CRC (2013).
+    </li>
+    <li>
+      Kasten, A., Scherp, A., Schauß, P.:
+      [[[eswc2014Kasten]]].
+      In: Presutti, V., d’Amato, C., Gandon, F., d’Aquin, M., Staab, S., and Tordai, A. (eds.)
+      The Semantic Web: Trends and Challenges. pp. 146–160.
+      Springer International Publishing, Cham (2014).
+      <a href="https://doi.org/10.1007/978-3-319-07443-6_11">https://doi.org/10.1007/978-3-319-07443-6_11</a> [[eswc2014Kasten]].
+    </li>
+    <li>
+      Hogan, A.:
+      [[[Hogan-Canonical-RDF]]].
+      ACM Trans. Web. 11, 22:1-22:62 (2017).
+      <a href="https://doi.org/10.1145/3068333">https://doi.org/10.1145/3068333</a> [[Hogan-Canonical-RDF]].
+    </li>
+    <li>
+      Arnold, R., Longley, D.:
+      RDF Dataset Normalization.
+      Report submitted to the W3C Credentials Community Group mailing list (2020).
+      <a href="https://lists.w3.org/Archives/Public/public-credentials/2021Mar/att-0220/RDFDatasetCanonicalization-2020-10-09.pdf">https://lists.w3.org/Archives/Public/public-credentials/2021Mar/att-0220/RDFDatasetCanonicalization-2020-10-09.pdf</a>.
+    </li>
+    <li>
+      Tomaszuk, D., Glab, S., Turobos, F., Pawlik, T., Kuzinski, D., Sopek, M.:
```

Comment:
It seems that a few diacritics were removed when I processed it through Zotero. I have restored them.
```suggestion
      Tomaszuk, D., Głąb, S., Turoboś, F., Pawlik, T., Kuziński, D., Sopek, M.:
```